src/sentry/data_export/endpoints/data_export_details.py

```diff
@@ -25,21 +27,21 @@ def get(self, request, organization, **kwargs):
             return Response(status=404)
 
         try:
-            data_export = ExportedData.objects.get(id=kwargs["data_export_id"])
-            # Check data export permissions
-            if data_export.query_info.get("project"):
-                project_ids = map(int, data_export.query_info.get("project", []))
-                projects = Project.objects.filter(organization=organization, id__in=project_ids)
-                if any(p for p in projects if not request.access.has_project_access(p)):
-                    raise PermissionDenied(
-                        detail="You don't have access to some of the data this export contains."
-                    )
-            # Ignore the download parameter unless we have a file to stream
-            if request.GET.get("download") is not None and data_export._get_file() is not None:
-                return self.download(data_export)
-            return Response(serialize(data_export, request.user))
+            data_export = ExportedData.objects.get(id=data_export_id, organization=organization)
         except ExportedData.DoesNotExist:
             return Response(status=404)
+        # Check data export permissions
+        if data_export.query_info.get("project"):
+            project_ids = map(int, data_export.query_info.get("project", []))
+            projects = Project.objects.filter(organization=organization, id__in=project_ids)
+            if any(p for p in projects if not request.access.has_project_access(p)):
+                raise PermissionDenied(
+                    detail="You don't have access to some of the data this export contains."
```

Comment:
```suggestion
                    detail="You don't have permissions to some of the data this export contains."
```

Reply:
thanks! i changed it to permissions and fixed the grammar